examples/chat/enterprise/aws-bedrock-anthropic/_template.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "app",
```

Comment:
For some reason, when I do `shiny create` -> `Generative AI templates...` -> `*`, I don't see any templates to choose from. I'm guessing that's because `./examples` aren't included in `MANIFEST.in`?

Reply:
Oh shoot, yeah. This always gets me because it works if installed via `-e`

Reply:
It's worth considering if you'd want to move these into `shiny/templates`

Reply:
Yea, I'd be ok moving them to `shiny/templates`

Reply:
Done! I considered duplicating the files but decided to move them. I'm not sure if or where those paths were used, but we'll definitely need to do a pass on the docs site to update and match these changes.